bin/shell

```diff
@@ -39,17 +39,14 @@ do_bindmount /run /run
 do_bindmount /proc /proc
 do_bindmount /sys /sys
 do_bindmount /dev /dev
-do_bindmount /dev/pts /dev/pts
```

Comment:
This change looks unrelated to the description of the PR - is that an accident or something that should be included here? If the later, a short rational would be nice.